src/Marten/Exceptions/MartenCommandExceptionFactory.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Marten.Services;
+using Npgsql;
+
+namespace Marten.Exceptions
+{
+    /// <summary>
+    /// Class responsible for MartenCommandException creation.
+    /// Based on the exact command code and exception creates
+    /// generi MartenCommandException or specific Exception derived from it.
+    /// </summary>
```

Comment:
few tweaks as below:
Class responsible for creating MartenCommandException exception or exceptions derived from it based on exact command code

Reply:
Updated.